testing-ui/src/main/java/com/bumble/appyx/testing/ui/rules/AppyxTestRule.kt

```diff
@@ -16,6 +16,7 @@ open class AppyxTestRule<T : Node>(
     private val launchActivity: Boolean = true,
     private val composeTestRule: ComposeTestRule = createEmptyComposeRule(),
     /** Add decorations like custom theme or CompositionLocalProvider. Do not forget to invoke `content()`. */
+    @Suppress("unused")
```

Comment:
unsure about whether to remove this variable or not

Reply:
Keep it and create an issue, I forgot to use it.